src/scripts.js

```diff
@@ -1 +1,53 @@
-console.log('Hello world');
+//query selectors
+const recipeCard = document.querySelector(".main-page");
+const username = document.querySelector(".username");
+let cookbook;
+let user;
+
+//event listeners
+window.onload = loadPage();
+
+//event handlers
+function loadPage() {
+  generateRandomUser();
+  generateCookbook();
+  displayAllRecipes();
+};
+
+//other functions
+function getRandomIndex(array) {
+  return Math.floor(Math.random() * array.length);
+};
+
+function generateRandomUser() {
+  let index = getRandomIndex(usersData);
+  user = new User(usersData[index], ingredientsData);
+  displayUsername();
+};
+
+function generateCookbook() {
+  cookbook = new Cookbook(recipeData, ingredientsData);
+};
+
+//DOM manipulation
+function displayUsername() {
+  username.innerText = `Username: ${user.name}`;
+};
+
+function displayAllRecipes() {
```

Comment:
This is awesome to get all of the recipe names to display! Next, we need to get the recipe image to display as well!